lib/java/com/google/android/material/bottomsheet/BottomSheetBehavior.java

```diff
@@ -1065,6 +1065,29 @@ public int getExpandedOffset() {
         : Math.max(expandedOffset, paddingTopSystemWindowInsets ? 0 : insetTop);
   }
 
+  /**
+   * Calculates the current offset of the bottom sheet.
+   *
+   * This method should be called when the child view is laid out.
+   *
+   * @return The offset of this bottom sheet within [-1,1] range. Offset increases
+   * as this bottom sheet is moving upward. From 0 to 1 the sheet is between collapsed and
+   * expanded states and from -1 to 0 it is between hidden and collapsed states. Returns
+   * {@code Float.MIN_VALUE} if the bottom sheet is not laid out.
+   */
+  public float calculateSlideOffset() {
```

Comment:
getCurrentSlideOffset()?

Reply:
The calculation does work, so I feel like calculate is more appropriate.

Reply:
I guess it's more about our API convention. But feel free to leave it as-is. I'll see what our team will say in the internal review. : )